opty/direct_collocation.py

```diff
@@ -727,6 +729,77 @@ def __init__(self, equations_of_motion, state_symbols,
             If True, STDOUT and STDERR of the Cython compilation call will be
             shown.
 
+        Attributes
+        ----------
+        Some of the attributes are explained in more detail under Parameters above.
+
+        eom: sympy.Matrix
```

Comment:
```suggestion
        eom: sympy.Matrix, shape(2*n, 1)
```

It is helpful to add shape information to all Matrices and/or arrays.